src/com/xilinx/rapidwright/util/MakeHWCTFromImpl.java

```diff
@@ -0,0 +1,191 @@
+/*

```

Comment:
This class seems specific for a particular application.  Could this be distributed outside of RapidWright?

Reply:
This will be used with PYNQ (I want to update OpenDFX to use this as well). I were thinking that if this is a part of RW, downloading rw_api jar and just call it. If this is distributed with PYNQ, does it means PYNQ have to has a full fork of RW? Once we are closing on the option, we need feedback from the PYNQ team.

Eddie and I agreed on that we will splits this PR after we clean up all comments.